packages/front-end/components/Radix/PremiumCallout.tsx

```diff
@@ -12,7 +12,7 @@ import Link from "@/components/Radix/Link";
 import styles from "./RadixOverrides.module.scss";
 
 export type Props = {
-  commercialFeature: CommercialFeature;
+  commercialFeature: CommercialFeature | null;
```

Comment:
I think we should revert all the changes to this file.  I want to keep this component purely for premium features.